docker-compose.yml

```
@@ -10,4 +10,6 @@ services:
       - .env
     volumes:
       - ./:/app
+      - ./docker-compose.yml:/app/docker-compose.yml:ro
+      - ./Dockerfile:/app/Dockerfile.yml:ro
```

Typo: `/app/Dockerfile.yml` ;)

To prevent docker from creating a folder when a file does not exist, use a bind mount:
```suggestion
      - type: bind
        source: ./docker-compose.yml
        target: /app/docker-compose.yml
        read_only: true
      - type: bind
        source: ./Dockerfile
        target: /app/Dockerfile
        read_only: true
```

I hope that this can be combined with the existing `./:/app` mount.

---

The relative paths (./docker-compose.yml) should create bind mounts.

So, I believe these are effectively the same thing:

```
volumes:
    - ./docker-compose.yml:...
```

and 
```
volumes:
   - type: bind
      source: ./docker-compose.yml
      ...
 ```